docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3.4'
+
+services:
+  ui_dev:
+    restart: always
+    image: better-chat-gpt
+    build:
+      context: ./
+    ports:
+      - 3000:3000
```

Comment:
The default port should be `5173` instead of `3000`

Reply:
I've changed the default port to be `5173`.